src/app/appActions.js

```diff
@@ -1,53 +1,50 @@
 import Promise from 'bluebird';
 import fetch from 'isomorphic-fetch';
-import { setLocaleMetadata } from '../helpers/metadata';
-
-export const GET_LOCALE = '@app/GET_LOCALE';
-
-export const SET_LOCALE = '@app/SET_LOCALE';
-export const SET_LOCALE_START = '@app/SET_LOCALE_START';
-export const SET_LOCALE_SUCCESS = '@app/SET_LOCALE_SUCCESS';
-export const SET_LOCALE_ERROR = '@app/SET_LOCALE_ERROR';
 
 export const GET_TRENDING_TOPICS = '@app/GET_TRENDING_TOPICS';
 export const GET_TRENDING_TOPICS_START = '@app/GET_TRENDING_TOPICS_START';
 export const GET_TRENDING_TOPICS_SUCCESS = '@app/GET_TRENDING_TOPICS_SUCCESS';
 export const GET_TRENDING_TOPICS_ERROR = '@app/GET_TRENDING_TOPICS_ERROR';
 
+export const GET_REWARD_FUND = '@app/GET_REWARD_FUND';
```

Comment:
I see that we are having to recreate these action types a lot, it might be good to have a helper like what I did here: https://github.com/busyorg/busy/blob/27c72a602096ff802996f0aef30a912fc1e750b8/src/helpers/stateHelpers.js#L106

We can probably go back and refactor the other action types in a seperate PR, keep the lines of code in each file smaller too